App/services/kdf.service.tsx

```diff
@@ -0,0 +1,8 @@
+import argon2 from 'react-native-argon2'
```

Comment:
FYI: When uploading to the Apple or Google play store the app gets put on US servers. When a person downloads the app to install they are technically "exporting" the app from the USA. As a result, we're subject to cryptography export restrictions. This can be easy or complicated for a number of reasons, one being if you use standard crypto algorithms (NIST certified example) or bespoke roll-your-own solutions. We may collectively need to think through what it means to include Argon2 out of the box as it will likely have a notable impact on our regulatory process (ie. registering with the U.S. Department of Commerce’s Bureau of Industry and Security)

Reply:
Argon2 is already implemented in indy anyways
https://github.com/hyperledger/indy-sdk/blob/master/wrappers/ios/libindy-pod/Indy/Wrapper/IndyWallet.h

Reply:
ios/Media.xcassets/AppIcon.appiconset/Contents.json
ios/Media.xcassets/Contents.json

Those are the results of linting and formatting workflows done when committing the work, I don't think they should be ignored, they seems to have some formatting